scidocs/user_activity_and_citations.py

```diff
@@ -166,10 +166,10 @@ def make_run_from_embeddings(qrel_file, embeddings, run_file, topk=5, generate_r
             # output is in this format: [qid iter paperid rank similarity run_id]
             if sorted_dists[i][0] in added:
                 continue
-            if i < topk:
-                results.append([pid, '0', sorted_dists[i][0], '1', str(np.round(sorted_dists[i][1], 5)), 'n/a'])
-            else:
+            if i < len(sorted_dists) - topk:
                 results.append([pid, '0', sorted_dists[i][0], '0', str(np.round(sorted_dists[i][1], 5)), 'n/a'])
```

Comment:
Should this be '1'? Now both are '0'.

Reply:
check line 172. It is already adding '1' for topk results.

Reply:
Line 172 has this on my screen: `results.append([pid, '0', sorted_dists[i][0], '1', str(np.round(sorted_dists[i][1], 5)), 'n/a'])`

Reply:
that is right (see the 4th column). the format is according to this:
`query-id Q0 document-id rank score STANDARD`

Reply:
Oh right :)